rust/operator-binary/src/crd.rs

```diff
@@ -131,6 +136,57 @@ impl AutoTlsCa {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub enum TlsKeyGeneration {
+    Rsa {
+        #[schemars(schema_with = "TlsKeyGeneration::tls_key_length_schema")]
+        length: u32,
+    },
+}
+
+impl TlsKeyGeneration {
+    pub const TLS_RSA_KEY_LENGTH_2048: u32 = 2048;
```

Comment:
optional: For me personally we don't need this constants, but feel free to leave them in!

Reply:
We could think about making this an explicit enum instead.

Reply:
Yeah i like it better like that

Reply:
What did we decide on here?